src/pymor/algorithms/pod.py

```diff
@@ -2,16 +2,19 @@
 # Copyright 2013-2019 pyMOR developers and contributors. All rights reserved.
 # License: BSD 2-Clause License (http://opensource.org/licenses/BSD-2-Clause)
 
+import numpy as np
+
+from pymor.algorithms.gram_schmidt import gram_schmidt
 from pymor.algorithms.svd_va import method_of_snapshots, qr_svd
 from pymor.core.defaults import defaults
 from pymor.core.logger import getLogger
 from pymor.operators.interfaces import OperatorInterface
 from pymor.vectorarrays.interfaces import VectorArrayInterface
 
 
-@defaults('rtol', 'atol', 'l2_err', 'method')
+@defaults('rtol', 'atol', 'l2_err', 'method', 'orth_tol')
 def pod(A, product=None, modes=None, rtol=4e-8, atol=0., l2_err=0.,
-        method='method_of_snapshots', svd_params=None):
+        method='method_of_snapshots', orth_tol=np.inf):
```

Comment:
I think I would prefer to let `orth_tol` to default to `1e-10`. That would correspond to the current behavior.

Reply:
Makes sense. I changed it.